src/components/forms/AutoComplete/index.tsx

```diff
@@ -1,74 +1,146 @@
-import React, {useEffect, useState} from "react";
-import { TextField } from "@material-ui/core";
+import React, { useEffect, useState } from 'react';
+import { TextField } from '@material-ui/core';
 import Autocomplete from '@material-ui/lab/Autocomplete';
-import Utils from "../../../helpers/utils";
-import TextInput from "../TextInput";
+import Utils from '../../../helpers/utils';
+import TextInput from '../TextInput';
 import isDeepEqual from 'fast-deep-equal/react';
 
+declare const PCore;
 
 interface IOption {
   key: string;
   value: string;
 }
 
-export default function AutoComplete(props) {
+export const getDisplayFieldsMetaData = columns => {
```

Comment:
I'm not sure why this function is outside the Autocomplete function scope. It doesn't seem to be used by other components. Should it be inside Autocomplete?

Reply:
moved the above methods inside Autocomplete function.